Makefile.build

```diff
@@ -4,11 +4,11 @@ LDFLAGS ?=
 
 ##@ Building
 .PHONY: build
-build:  ## build all applications
-	go build -ldflags "$(LDFLAGS)" -o $(BIN_DIR)/block-explorer cmd/block-explorer/*.go
+build: block-explorer api loadtest_migrate## build all applications
```

Comment:
I run `make build` and api binary don't creates:
```
go build -ldflags "" -o bin/block-explorer cmd/block-explorer/*.go
go build -ldflags "" -o bin/loadtest_migrate cmd/loadtest_migrate/*.go
go build -ldflags "" -o bin/migrate cmd/migrate/*.go
```

Reply:
I also think you missed one of my comments from friday (its for full PR, not for specific line, check Conversation tab in PR to see it)